lib/internal/util.js

```diff
@@ -362,15 +362,14 @@ function getConstructorOf(obj) {
  * @returns {string} The current working directory or the volume root if it cannot be determined.
  */
 function getCwdSafe() {
-  let cwd;
+  const { sep } = require('path');
+  let cwd = '';
 
   try {
-    cwd = process.cwd() + '/';
-  } catch {
-    cwd = '/';
-  }
+    cwd = process.cwd();
+  } catch {}
```

Comment:
The `/**/` from Jordan's suggestion was needed to appease the linter.

```suggestion
  } catch {/**/}
```

```suggestion
  } catch {} // eslint-disable-line no-empty
```